src/System.IdentityModel.Tokens.Jwt/JwtSecurityToken.cs

```diff
@@ -469,6 +469,18 @@ public override string ToString()
                 return Header.SerializeToJson() + ".";
         }
 
+        /// <inheritdoc/>
+        public override string SecureToString()
```

Comment:
>SecureToString [](http://example.com/codeflow?start=31&length=14)

Why not simply call these what they do? 
NoSignatureToString() and WithSignatureToString()

Reply:
For that matter why such a switch: LogHelper.LogTokensWithSignature.